docs/app/Examples/elements/List/Types/ListListExample.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListListExample extends Component {
+  render() {
+    return (
+      <List>
+        <ListItem description='Apples' />
+        <ListItem description='Pears' />
+        <ListItem description='Oranges' />
+      </List>
+    );
+  }
+}
```

Comment:
I see what you mean.  It looks like they just show a mashup of various lists at the top, then dive into each specific capability.  I think we're good with the basics up top if we have links/icons/etc down below.